<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>948</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_navigate" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>278</width>
      <height>861</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>278</width>
          <height>861</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Data source</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
            <item>
             <widget class="QTabWidget" name="tab_src">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab_3">
               <attribute name="title">
                <string>Webcam</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_7">
                    <property name="text">
                     <string>Camera num :</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spin_camera_num">
                    <property name="minimum">
                     <number>0</number>
                    </property>
                    <property name="maximum">
                     <number>5</number>
                    </property>
                    <property name="value">
                     <number>0</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="cb_sensor">
                  <property name="text">
                   <string>Show sensor signal</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_4">
               <attribute name="title">
                <string>Numpy</string>
               </attribute>
               <layout class="QGridLayout" name="gridLayout_4">
                <item row="1" column="1">
                 <widget class="QPushButton" name="b_np_load_signal">
                  <property name="text">
                   <string>Load signal</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="b_np_load_video">
                  <property name="text">
                   <string>Load video</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="lbl_np_signal_name">
                  <property name="text">
                   <string>None</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="lbl_np_video_name">
                  <property name="text">
                   <string>None</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="2">
                 <widget class="QCheckBox" name="cb_reference">
                  <property name="text">
                   <string>Show reference signal</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_5">
               <attribute name="title">
                <string>Video File</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="lbl_file_video_name">
                  <property name="text">
                   <string>None</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="b_file_load_video">
                  <property name="text">
                   <string>Load video</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>FPS</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="spin_fps">
                <property name="minimum">
                 <number>5</number>
                </property>
                <property name="maximum">
                 <number>60</number>
                </property>
                <property name="value">
                 <number>30</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="b_start">
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTabWidget" name="tab_mode">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>BPM</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="4">
             <widget class="QCheckBox" name="cb_bpm_55">
              <property name="text">
               <string>55</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="cb_bpm_35">
              <property name="text">
               <string>35</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QCheckBox" name="cb_bpm_45">
              <property name="text">
               <string>45</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="cb_bpm_5">
              <property name="text">
               <string>5</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QCheckBox" name="cb_bpm_50">
              <property name="text">
               <string>50</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="6">
             <widget class="QPushButton" name="b_show_bpm">
              <property name="text">
               <string>Show</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QCheckBox" name="cb_bpm_30">
              <property name="text">
               <string>30</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="cb_bpm_15">
              <property name="text">
               <string>15</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QCheckBox" name="cb_bpm_20">
              <property name="text">
               <string>20</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="cb_bpm_40">
              <property name="text">
               <string>40</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QCheckBox" name="cb_bpm_25">
              <property name="text">
               <string>25</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cb_bpm_10">
              <property name="text">
               <string>10</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="tristate">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QCheckBox" name="cb_bpm_60">
              <property name="text">
               <string>60</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_estimated" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>640</width>
      <height>150</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="7,3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_signal_estimated">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="7,4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="lbl_bpm_estimated">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>72</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8" stretch="4,3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_19">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
           </property>
           <property name="text">
            <string>Remote</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_18">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
           </property>
           <property name="text">
            <string>bpm</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_reference" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>640</width>
      <height>150</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="7,3">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lbl_signal_reference">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="7,4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="lbl_bpm_reference">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>72</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9" stretch="4,3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="label_21">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
           </property>
           <property name="text">
            <string>Reference</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_20">
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 255, 0);</string>
           </property>
           <property name="text">
            <string>bpm</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_status" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="lbl_fps">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lbl_fps_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>fps</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
